samples/client/petstore/scala-sttp/build.sbt

```diff
@@ -2,23 +2,17 @@ version := "1.0.0"
 name := "scala-sttp-petstore"
 organization := "org.openapitools"
 
-scalaVersion := "2.13.0"
-
-crossScalaVersions := Seq(scalaVersion.value, "2.12.10",  "2.11.12")
+scalaVersion := "2.13.2"
+crossScalaVersions := Seq(scalaVersion.value, "2.12.10")
 
 libraryDependencies ++= Seq(
-  "com.softwaremill.sttp.client" %% "core" % "2.0.0",
-  "com.softwaremill.sttp.client" %% "json4s" % "2.0.0",
-  "org.json4s" %% "json4s-jackson" % "3.6.7",
-  // test dependencies
-  "org.scalatest" %% "scalatest" % "3.0.8" % Test,
-  "junit" % "junit" % "4.13" % "test"
+  "com.softwaremill.sttp.client" %% "core" % "2.2.0",
+  "com.softwaremill.sttp.client" %% "json4s" % "2.2.0",
+  "org.json4s" %% "json4s-jackson" % "3.6.8"
 )
 
 scalacOptions := Seq(
   "-unchecked",
   "-deprecation",
   "-feature"
 )
-
-publishArtifact in (Compile, packageDoc) := false
```

Comment:
We should consider adding this back. Default behavior should be to avoid accidentally publishing content. Users can then configure publishing and ignore the build file, or enable published at invocation.

Reply:
Right, well our goal here was to create a minimal working project, keeping things as simple as possible, usable for an application developer (who doesn't normally publish artifacts, or has a custom publishing pipeline anyway). But maybe our approach here is incorrect - are there some guidelines as to what should be the functionality of the generated build? 

Thanks!